_toc.yml

```diff
@@ -6,29 +6,24 @@ parts:
       - file: notebooks/how-to-cite
   - caption: Introduction to UXarray & Unstructured Grids
     chapters:
-      - file: notebooks/01-intro/unstructured-grid-overview
-      - file: notebooks/01-intro/data-structures
-      - file: notebooks/01-intro/plot-api
+      - file: notebooks/01-intro/01-unstructured-grid-overview
+      - file: notebooks/01-intro/02-data-structures
+      - file: notebooks/01-intro/03-plot-api
 
-  - caption: Methods for Unstructured Grid Visualization
+  - caption: Methods & Libraries for Unstructured Grid Visualization
     chapters:
-      - file: notebooks/02-methods/template
+      - file: notebooks/02-methods/01-data-mapping
```

Comment:
Capitalize the term "Mapping" from the top level markdown block (#) in this notebook

Reply:
Done